builder/hyperone/examples/basic.pkr.hcl

```diff
@@ -0,0 +1,25 @@
+
+variable "project" {
```

Comment:
It might be a good idea to add a required_plugin block to these example packer hcl files, unless this file will also work on the v1 API